### ISSUE TITLE
[steam community] filechooser initial path

### ISSUE THREAD

**Issue by mx1up** [ValveSoftware/steam-for-linux#1907]
Hi,

I have the following issue when using the filechooser in steam, for example, to upload an image for creating a guide:

the filechooser initially starts in ~/.local/share/Steam install dir but does not show this on top. also, it's impossible to navigate. You get out of it by either selecting a file there and pressing the filechooser button again (then it is correctly initialized). Or you can type /home/youruser and press open and it will reconfigure itself correctly.

Bottom line: the path initialisation of the filechooser fails somewhere.
Either way, the filetype combobox stays empty.

See the attached screenshot for clarification
![steam_fc1](https://f.cloud.github.com/assets/178714/174019/741a1f06-7af2-11e2-8a78-033bdccfc979.png)

**Comment by eiglow** [comment]
Not sure if this issue still exists, I was able to navigate immediately after opening the filechooser. The filetype combobox is still empty, which meant I could pick random non-image files for a guide's branding image, for example, but I guess that's a separate issue.
